### PR TITLE
Feature bridging

### LINKED ISSUE #138: NsxLogicalRouter bridging function support
NSX DLR bridging function is not yet supported in powernsx

### PR DESCRIPTION
**Pull request by nmbradford**
Fixes #138

### CODE REVIEW

File: module/PowerNSX.psm1 [hunk 1]
```diff
@@ -21300,6 +21352,397 @@ function New-NsxLogicalRouterRedistributionRule {
...
+    instance.  A Bridge is configured between a single VD Port Group and a single 
+    Logical Switch
+
+    Each Logical Router can define the configuration of multiple bridges.
+
+    The New-NsxLogicalRouterBridge cmdlet creates a new bridge instance configured
+    via the specifid logical router.
+    
+    .EXAMPLE
+    Get-NsxLogicalRouter BridgeRouter | Get-NsxLogicalRouterBridging | New-NsxLogicalRouterBridge -Name "bridge1" -PortGroup $bridgepg1 -LogicalSwitch $bridgels1
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: add before $bridgepg1 = get-NSX... ?

File: module/PowerNSX.psm1 [hunk 2]
```diff
@@ -21300,6 +21352,397 @@ function New-NsxLogicalRouterRedistributionRule {
...
+    Logical Switch
+
+    Each Logical Router can define the configuration of multiple bridges.
+    
+    The Get-NsxLogicalRouterBridge cmdlet retrieves the bridge instances configured
+    within the specified LogicalRouter Bridging Configuration.
+
+    .EXAMPLE
+    Get-NsxLogicalRouter LogicalRouter01 | Get-NsxLogicalRouterBridging | Get-NSxLogicalRouterBridge
+
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: missing text for example

File: tests/integration/03.LogicalRouter.Tests.ps1 [hunk 3]
```diff
@@ -163,6 +169,59 @@ Describe "Logical Routing" {
...
+        }
+
+        It "Can create a bridge instance" {
+            $bridge = Get-NsxLogicalRouter $name | Get-NsxLogicalRouterBridging | New-NsxLogicalRouterBridge -Name "pester_bridge_1" -PortGroup $BridgePortGroup1 -LogicalSwitch $bridgels1
+            $bridge | should not be $null
+            $Bridge.Name| should be "pester_bridge_1"
+        }
+
+        It "Can retrieve a bridge instance by name" {
+            $null = Get-NsxLogicalRouter $name | Get-NsxLogicalRouterBridging | New-NsxLogicalRouterBridge -Name "pester_bridge_1" -PortGroup $BridgePortGroup1 -LogicalSwitch $bridgels1
```
Comment (alagoutte) [inline tests/integration/03.LogicalRouter.Tests.ps1]: use variable name for -Name ?

**Comment by nmbradford** [comment]
Tests passing - still environment issues impacting 631 tests.

**Pull request merged**